src/telemetryReporter.ts

```diff
@@ -65,6 +66,8 @@ export default class TelemetryReporter extends vscode.Disposable {
         }
 
         this.appInsightsClient.commonProperties = this.getCommonProperties();
+        this.appInsightsClient.context.tags[this.appInsightsClient.context.keys.userId] = vscode.env.machineId;
```

Comment:
Wrap both these lines with an `if (vscode && vscode.env)` as we do in the `getCommonProperties()`

Reply:
Sounds good.